plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -7967,13 +7967,53 @@ public void testNonSelectStatementsWithPartitionFilterRequired()
         assertUpdate(session, "DROP TABLE " + tableName2);
     }
 
+    @Test
+    public void testPartitionFilterRequiredSchemas()
+    {
+        String schemaName = "gold_mart";
```

Comment:
Append `randomNameSuffix()` and rename to more meaningful name, e.g. `test_unenforced_schema`